pkg/operator/operator.go

```diff
@@ -84,13 +78,35 @@ func NewOpenShiftControllerManagerOperator(
 	return c
 }
 
+// getOperatorConfig execute get request using operator config client.
+func (c OpenShiftControllerManagerOperator) getOperatorConfig() (
+	*operatorapiv1.OpenShiftControllerManager, error,
+) {
+	return c.operatorConfigClient.OpenShiftControllerManagers().
+		Get(context.TODO(), "cluster", metav1.GetOptions{})
+}
+
+// updateStatus execute update status request against operator config client, when modified.
+func (c OpenShiftControllerManagerOperator) updateStatus(
+	original *operatorapiv1.OpenShiftControllerManager,
+	modified *operatorapiv1.OpenShiftControllerManager,
+) (*operatorapiv1.OpenShiftControllerManager, error) {
+	if equality.Semantic.DeepEqual(original.Status, modified.Status) {
+		return modified, nil
+	}
+	opts := metav1.UpdateOptions{}
+	return c.operatorConfigClient.OpenShiftControllerManagers().
+		UpdateStatus(context.TODO(), modified, opts)
```

Comment:
why declare opts ... just pass metav1.UpdateOptions{} as a parameter

Reply:
Was just a left over from a previous interaction. In general, that's a way to reduce line width.